src/server.rs

```diff
@@ -13,6 +13,8 @@ use futures::{
 use miette::IntoDiagnostic as _;
 use tonic::{Request, Response, Status};
 
+use crate::convert;
+pub use crate::validator::Validator;
```

Comment:
This should be after the `use crate::{...}` import

Reply:
This is just not me knowing precisely what I'm doing... Fixed, thanks!